src/main/resources/lang.yml

```diff
@@ -5,6 +5,7 @@ lang:
     INVALIDARGS: '§c§OInvalid command arguments, expecting: §7§O%s'
     SPAM: §8§OToo many requests at once
     INVENTORYFULL: §c§OYour inventory is full
+    PLAYERONLY: Player only command
```

Comment:
Should we have a more detailed and formatted error message here?

Reply:
ACF seems to add some formatting to the exception thrown in the Context, it ends up being a red Error: Player only command, we could definitely change the text, idk what to though